config/routes.rb

```diff
@@ -4,4 +4,6 @@
   namespace :auth do
     post 'facebook/callback', to: 'facebook_callbacks#create'
   end
+
+  resources :targets
```

Comment:
Be careful with this, right now you allow `create, edit, destroy & update`, you just want `create` right now